tools/reviewer/job/tasks/ReviewerMetadataUpdaterTask.java

```diff
@@ -153,5 +163,148 @@ public void run() {
   public Boolean shouldRun() {
     return !lock.isLocked();
   }
+
+  public ReviewerConfig getReviewerConfig(String filePath) throws IOException {
+    ReviewerConfig reviewerConfig =
+        (ReviewerConfig) fileUtils.readPrototxt(filePath, ReviewerConfig.newBuilder());
+    return reviewerConfig;
+  }
+
+  public String getStartupOsReviewerConfigPath() {
+    String localRegistryFilePath =
+        fileUtils.joinPaths(fileUtils.getCurrentWorkingDirectory(), "reviewer_config.prototxt");
+    return localRegistryFilePath;
+  }
+
+  public String getHasadnaReviewerConfigPath() {
+    String targetDirectory = fileUtils.expandHomeDirectory("~/hasadna");
+    String localHasadnaRegistryFilePath =
+        fileUtils.joinPaths(targetDirectory, "reviewer_config.prototxt");
+    return localHasadnaRegistryFilePath;
+  }
+
+  private User getUser(ReviewerConfig reviewerConfig, String userId) {
+    for (User user : reviewerConfig.getUserList()) {
+      if (user.getId().equals(userId)) {
+        return user;
+      }
+    }
+    return User.getDefaultInstance();
+  }
+
+  public void compareReviewerConfigData(
+      ReviewerConfig reviewerConfig1, ReviewerConfig reviewerConfig2) {
+    final String displayName = reviewerConfig1.getDisplayName();
+    LinkedHashSet<Repo> repoList = new LinkedHashSet<>();
+    // Getting ReviewerConfig1's repos
```

Comment:
I'd advise removing unnecessary comments. It's good practice to use comments for difficult code or for explaining some things. It's also fine to use it in the developing stage. 
But the production code should contain only really need comments. Please filter the comments and remove unnecessary ones.

Reply:
Should I keep these comments?
<if (user2.getId().isEmpty()) {
        // (user defined only in reviewerConfig1). Add `user1` to `mergedUsersList` without changes
        mergedUsersList.add(user1);
      } else {
        // (users defined in both repos). Merge data and add merged result to `mergedUsersList`>

Reply:
I think you can leave one comment:
`// User defined in both repos. Merging data.`

Reply:
Done.